content/getting-started/guides/get-archives-tutorial/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+weight: 30
+category: "getting-started"
+subcategory: "guides"
+title: "Using API in practice"
+tagline: "Using LiveChat Messaging API in practice."
+desc: "Learn how to get chat summaries using our API. A step-by-step tutorial that guides you through the process."
+---
+
+If you're not sure how to use our APIs, this tutorial will guide you through the process of sending a sample request. We'll cover all necessary aspects including authorization, the request format, as well as recommended tools.
+To demonstrate the process, we'll show you how to **get chat summaries**.
+
+In short, there are 3 main steps:
+
+1. Creating an app in Developer Console
+
+2. Acquiring an access token
+
+3. Sending a request
+
+
+# Getting started
+
+Before you start, make sure you have a <a href="https://accounts.livechatinc.com" target="_blank">LiveChat account</a>.
+
+# Step 1 Creating an app in Developer Console
+
+Start by logging in to <a href="https://developers.livechatinc.com/console/" target="_blank">Developer Console</a> and [create a new server-side app](../#creating-livechat-apps). The **Authorization** building block is added automatically.
+
+![LiveChat New App](images/create_app.png)
+
+Next, fill in the **Redirect URI whitelist** field and choose scopes. 
+
+To get chat sumamries, you'll need `chats--all:ro` and `chats--access:ro`, and `chats--my:ro`. You can check out method details in our [API documentation](../../../messaging/agent-chat-api/#get-chats-summary).
+
+![LiveChat Developer Console](images/console.png)
+
+
+When it's all set, we can proceed with token generation. Make sure to have your **Client Id** and **Redirect URI** at hand.
+
+# Step 2 Acquiring the access token
+
+Everything you need to know about authorization can be found in [this document](../../../getting-started/authorization), so keep this source for future reference.
+
+Start by pasting your **Client Id** and **Redirect URI** in the following request:
+
+```shell
+https://accounts.livechatinc.com/
+?response_type=token
+&client_id=<Client_Id_from_Developer_Console>
+&redirect_uri=<Redirect_URI_from_Developer_console>
+```
+
+Given that your **Client Id** was **0805e283233042b37f460ed8fbf22160** and **Redirect URI** was **https://my.livechatinc.com/**, this is how your request will look: 
+
+```shell
+https://accounts.livechatinc.com/
+?response_type=token
+&client_id=0805e283233042b37f460ed8fbf22160
+&redirect_uri=https://my.livechatinc.com/
+```
+
+Paste your request in the browser and don't miss the response. 
+
+![LiveChat request](images/request.png)
+
+Your access token will be returned in a long string:
+
+```
+https://my.livechatinc.com/
+#access_token=dal%3ALW9Guwt8QMiPK6P6G8cSDw
+&expires_in=28800
+...
+```
+
+![LiveChat response](images/response.png)
```

Comment:
This should exactly explain which part of the link is the token, also that "%3A" is URL-coded part for ":"